src/test/scala/beam/router/TransitInitializerSpec.scala

```diff
@@ -18,7 +18,7 @@ class TransitInitializerSpec extends WordSpecLike with Matchers with MockitoSuga
     val transitInitializer: TransitInitializer = init
 
     "return SUV, based on agency[217] and route[1342] map" in {
```

Comment:
If this is temporary then that's fine - otherwise the test description should probably be changed also :)